.github/workflows/deploy-tag.yml

```diff
@@ -43,9 +43,23 @@ jobs:
       id-token: write
     timeout-minutes: 70
     needs: deploy
+    outputs:
+      base64-subjects: ${{ steps.attest.outputs.base64-subjects }}
     steps:
       - name: Attest
-        uses: johnbillion/action-wordpress-plugin-attestation@0.4.0
+        id: attest
+        uses: johnbillion/action-wordpress-plugin-attestation@slsa
         with:
           version: ${{ needs.deploy.outputs.version }}
           zip-path: ${{ needs.deploy.outputs.zip-path }}
+          dry-run: true
+  slsa:
+    name: Generate SLSA provenance attestation
+    needs: attest
+    permissions:
+      contents: read
```

Comment:
```suggestion
      contents: write
```

Reply:
Annoying that the workflow needs to be granted write permission even though the condition to run that job isn't satisfied.